README.md

```diff
@@ -0,0 +1 @@
+# Add linter for HTML and CSS
```

Comment:
- [ ] Kindly use the microverse readme template for adding a professional readme to your projects. Keep in mind that if you don't do this in future projects your submission will be marked as invalid. 

Feel free to edit that template so that it contains all the details about your project.

[Microverse readme template](https://github.com/microverseinc/readme-template)

Reply:
@GiuseppeTG thank you for the review, I have made the changes as advised. How can I request another review?

Reply:
@SaneMyburg You did it well. So, when you are requested changes you must do the changes, commit, and push again, same branch and same PR. When it is approved you can merge it. The point of this is that reviewers need to see also previous reviewers work in your pull request (: